src/index.ts

```diff
@@ -40,21 +53,66 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('hasOperationSupportingMultipart', function(value: CodegenOperation | CodegenOperation[]): boolean {
```

Comment:
We can rename this to `operationSupportsMultipart` and it no longer needs to take an array parameter.